.yamato/com.unity.ml-agents-test.yml

```diff
@@ -1,7 +1,11 @@
 test_editors:
   - version: 2018.4
+    # 2018.4 doesn't support code-coverage
```

Comment:
The dev project is on 2019.3 anyway, is there really a need to have a line for 2018.4?

Reply:
Yes, this doesn't run the dev project; I believe the `upm-ci package test` creates its own temporary test project. 

You can see one of the failed 2018.4 test runs here: https://yamato.prd.cds.internal.unity3d.com/jobs/497-ml-agents-develop/tree/develop-coverage-yamato/.yamato%252Fcom.unity.ml-agents-test.yml%2523test_mac_2018.4/1611365/job